### PR TITLE
1507 sso for builtin mechanisms

### PR DESCRIPTION
**Pull request by teddyjtorres**
For issue #1507 .

### CODE REVIEW

**Comment by teddyjtorres** [comment]
#build

**Comment by teddyjtorres** [comment]
There are opportunities to refactor the Basic, Form, and CustomForm mechanisms into a common framework.  That work can be done in another commit in a separate PR or here.

File: dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -98,3 +98,9 @@ contextRootForFormAuthenticationMechanism.desc=Specifies the context root of a f
...
+
+jaspicSessionForMechanismsEnabled=Enable JASPIC session single sign-on
+jaspicSessionForMechanismsEnabled.desc=Specifies whether JASPIC session single sign-on is enabled for the container HttpAuthenticationMechanism bean implementations for the BasicAuthenticationMechanismDefinition, FormAuthenticationMechanismDefinition, or CustomFormAuthenticationMechanismDefinition annotations.
+
+jaspicSessionCookieName=JASPIC session single sign-on cookie name
+jaspicSessionCookieName.desc=Customizes the JASPIC session SSO cookie name. A custom cookie name allows you to logically separate authentication between JASPIC session SSO domains and to enable customized authentication to a particular environment. Before setting this value, consider that setting a custom cookie name can cause an authentication failure. For example, a connection to a server that has a custom cookie property set sends this custom cookie to the browser. A subsequent connection to a server that uses either a different cookie name, is not able to authenticate the request via a validation of the in-bound cookie.
```
Comment (cjcarpen) [inline dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties]: jaspicSessionCookieName=JASPIC session single sign-on cookie name
jaspicSessionCookieName.desc=Customizes the JASPIC session single sign-on (SSO) cookie name. A custom cookie name enables you to logically separate authentication between JASPIC session SSO domains and to enable customized authentication to a particular environment. Before you set this value, consider that setting a custom cookie name can cause an authentication failure. For example, a connection to a server that has a custom cookie property set sends this custom cookie to the browser. A subsequent connection to a server that uses a different cookie name is not able to authenticate the request by validating the inbound cookie.
**Check the edits to the last sentence here - I removed a few of what seemed to be extra words**

**Comment by teddyjtorres** [comment]
@cjcarpen Hi Carolyn. Thank you for the review. The description was modified as suggested.

**Comment by teddyjtorres** [comment]
#build

**Pull request merged**